gulpfile.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Is `gulpfile.js` required on top(root) directory?
Can you move it to `test/issues/24` dir?